src/utils/extractActionFromPath.js

```diff
@@ -1,5 +1,5 @@
 function extractActionFromPath(path) {
-  return path.split('/')[2];
+  return path.split('/')[2] || '';
```

Comment:
Is it really good idea? Returning undefined if it doesn't exist looks like good deal for me.